nodes/src/nodes/audio_tts/IGlobal.py

```
@@ -0,0 +1,115 @@
+                audio_arr = np.clip(audio_arr, -1.0, 1.0)
+                with wave.open(out_path, 'wb') as wavf:
+                    wavf.setnchannels(1)
+                    wavf.setsampwidth(2)
+                    wavf.setframerate(_KOKORO_SAMPLE_RATE)
+                    wavf.writeframes((audio_arr * _INT16_MAX).astype(np.int16).tobytes())
+            return {'path': out_path, 'mime_type': _WAV_MIME}
+        except Exception:
+            try:
+                os.remove(out_path)
+            except OSError:
+                pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)